modules/home/workstation/i3/polybar.lib.nix

```diff
@@ -0,0 +1,29 @@
+{ config, pkgs, lib, ... }:
+
+let
+  launchScript = ''
+    # Terminate already running bar instances
+    ${pkgs.procps}/bin/pkill polybar
```

Comment:
The reason why it's not getting started is that this is killing the script itself (aka `$0`). That's because `pkill` takes a pattern as opposed to taking a full name.

Quoting `man pkill`

```
       pattern
              Specifies an Extended Regular Expression for matching against the process names or command lines.
```

Reply:
Yes indeed. We should iterate trough everything `pgrep` finds and kill it only if the process name is not `polybar-script`

Reply:
Fixed locally